### PR TITLE
Update to flash_player_ppapi 27.0.0.130

### PR DESCRIPTION
**Pull request by sergiomb2**

### CODE REVIEW

File: flash-plugin.spec.in [hunk 1]
```diff
@@ -2,81 +2,55 @@
...
+Source0:        https://fpdownload.adobe.com/get/flashplayer/pdc/%{version}/flash_player_ppapi_linux.i386.tar.gz
+%else
+Source1:        https://fpdownload.adobe.com/get/flashplayer/pdc/%{version}/flash_player_ppapi_linux.x86_64.tar.gz
+%endif
 
-BuildRequires:  desktop-file-utils
 Requires:       mozilla-filesystem
+Requires:       freshplayerplugin
+Obsoletes:      flash-plugin-properties
+Obsoletes:      flash-player-ppapi
```
Comment (kwizart) [inline flash-plugin.spec.in:22]: Please obsoletes/provides with a version

Don't obsolete flash-player-ppapi in the npapi version

Comment (kwizart) [reply flash-plugin.spec.in:22]: There is no point to requires freshplayerplugin at all with the npapi plugin earch browser should use the related flash plugins (npapi for firefox, ppapi for chrome based).

File: flash-plugin.spec.in [hunk 2]
```diff
@@ -2,81 +2,55 @@
...
 URL:            http://get.adobe.com/flashplayer/
 ExclusiveArch:  %{ix86} x86_64
 
-Source0:        http://fpdownload.macromedia.com/get/flashplayer/pdc/%{version}/install_flash_player_11_linux.i386.tar.gz
-Source1:        http://fpdownload.macromedia.com/get/flashplayer/pdc/%{version}/install_flash_player_11_linux.x86_64.tar.gz
+%ifarch %{ix86}
+Source0:        https://fpdownload.adobe.com/get/flashplayer/pdc/%{version}/flash_player_ppapi_linux.i386.tar.gz
+%else
+Source1:        https://fpdownload.adobe.com/get/flashplayer/pdc/%{version}/flash_player_ppapi_linux.x86_64.tar.gz
+%endif
```
Comment (sergiomb2) [inline flash-plugin.spec.in:17]: I am downloading flash_player_ppapi_linux.x86_64.tar.gz , but we can do both packages 
flash_player_Npapi_linux here and other with flash_player_Ppapi_linux, is what you mean ?

**Comment by sergiomb2** [comment]
https://github.com/sergiomb2/lpf-flash_player_ppapi created, as lpf-flash_plugin stay provides flash_plugin, an option maybe do new package flash_player_ppapi that will work with fresh player package. 
Closing this PR

**Pull request closed without merging**